_includes/doc-index.html

```diff
@@ -26,27 +28,30 @@ <h1 id="qubes-os-documentation">{{ topic.topic }}</h1>
 <p> {{ topic.subtopic }}</p>
 {% assign english_count = 0 %}
 {% for item in ind %}
-  {% assign item_en = index_english[english_count] %}
-  {% assign english_count = english_count | plus: 1 %}
+  {% assign item_en = index_english[english_count] -%}
+  {% assign english_count = english_count | plus: 1 -%}
   <a id="{{ item_en.title | downcase | replace: " ", "-" | replace: ",", ""}}"></a>
   <h2 id="{{ item.title | downcase | replace: " ", "-" | replace: ",",""}}">{{ item.title }}</h2>
   {% if item.subtitle %}
     <p>{{ item.subtitle }}</p>
   {% endif %}
-  {% assign subpages = item.sub-pages | where_exp: "item", "item.title == nil" %}
-  {% assign subpages2 = item.sub-pages | where_exp: "item", "item.title != nil" %}
-  {% assign subsubpages_en = item_en.sub-pages | where_exp: "item", "item.title != nil" %}
+  {% assign subpages = item.sub-pages | where_exp: "item", "item.title == nil" -%}
+  {% assign subpages2 = item.sub-pages | where_exp: "item", "item.title != nil" -%}
+  {% assign subsubpages_en = item_en.sub-pages | where_exp: "item", "item.title != nil" -%}
   <ul>
-  {% for sub in subpages %}{% for page in site.doc %}{% if page.url == sub.url %}
-    <li><a href="{{ sub.url }}">{{ page.title }}</a></li>
-  {% endif %}{% endfor %}{% endfor %}
-  {% for sub in subpages %}{% if item.category == 'external' %}
-    <li><a href="{{ sub.url }}">{{ sub.text }}</a></li>
-  {% endif %}{% endfor %}
```

Comment:
I must have missed this during my latest cleanup, or I would have broken these operators up onto multiple lines.

(I originally put them all on the same line because I observed that it resulted fewer empty lines, but then I realized that we should find some better way to handle that than forcing such a change onto our source code formatting.)